content/docs/contributing/core.md

```diff
@@ -228,9 +228,9 @@ $ export AZURE_STORAGE_CONNECTION_STRING="DefaultEndpointsProtocol=http;AccountN
 > the Google account.
 
 To avoid tests flow interruption by manual login, perform authorization once and
-backup the obtained Google Drive access token, which is stored by default under
-`.dvc/tmp/gdrive-user-credentials.json`. Restore `gdrive-user-credentials.json`
-from backup for any new DVC repo setup to avoid manual login.
+DVC will automatically cache the obtained Google Drive access token in a global
+file (e.g. by default `~/.cache/pydrive2fs/$client_id/default.json` on Linux) to
+avoid manual login next time.
```

Comment:
```suggestion
DVC will automatically cache the Google Drive access token obtained in a global
location (e.g. `~/.cache/pydrive2fs` on Linux, see [details]) to avoid manual
logins after this.

[details]: https://dvc.org/doc/command-reference/remote/modify#google-drive
```

Reply:
Would be ideal to move the `[details]` ref link to the bottom of the section.